src/Utils.ps1

```diff
@@ -1,7 +1,28 @@
 $ModuleBasePath = "$PSScriptRoot\.."
 
-function setenv($key, $value) {
-    [void][Environment]::SetEnvironmentVariable($key, $value)
+function setenv {
+    param(
+        [Parameter()]
+        [string]
+        $key,
+
+        [Parameter()]
+        [string]
+        $value,
+
+        [Parameter()]
+        [ValidateSet("Process", "User")]
+        [string]
+        $Scope = "Process"
+    )
+
+    $type = [System.EnvironmentVariableTarget]::Process
```

Comment:
I'm reasonably certain you can just pass `$Scope` directly to `SetEnvironmentVariable()` and PS will automatically cast for you.

Reply:
@dahlbyk  You're right. I've removed the conversion.